PRESUBMIT.py

```diff
@@ -18,6 +19,21 @@
 # pylint: disable=line-too-long,protected-access
 
 
+# Use a custom git diff call instead of input_api.AffectedSourceFiles(). This
+# way we get non-staged files similar to how "git cl format" works.
```

Comment:
Perhaps we can explain why we want to consider non-staged files for formatting but not for lint (`CheckESLint` still only considers `input_api.AffectedFiles`)?

Reply:
I think if we trust eslint and it's `--fix` machinery in a sense that it won't trigger next presubmit Checks, then we can also use `git cl format`-like diff for `CheckESLint`. This way all `--fix`-aware checks will work similarly.

Reply:
Actually after further digging, I'm inclined to use the default presubmit diff logic, because it supports `--files` parameter, which means we can run prettier/eslint on arbitrary directories via `npm run presubmit -- --files=.../*` command.

The fact that `git cl format` uses a slightly different diff logic is not perfect, but we can live with that.